### PR TITLE
docs(uib): configure healthcheck in production

### PR DESCRIPTION
**Pull request by VishalThapaliya**
Covers [PROD-336](https://bonitasoft.atlassian.net/browse/PROD-336)

[PROD-336]: https://bonitasoft.atlassian.net/browse/PROD-336?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/applications/pages/ui-builder/production-packaging.adoc [hunk 1]
```diff
@@ -63,6 +63,37 @@ docker build --build-arg "BASE={bonita-ui-builder-image}" \
              -t <image-name>:<version> .
 ----
 
+=== Configure and manage Bonita healthcheck settings in a production environment
+
+The Bonita healthcheck is a mechanism used to monitor the health and availability of Bonita services in production environments.
+It periodically checks the status of these services to ensure they are functioning properly, helping to identify issues early and maintain system stability.
+
+Changing the default credentials for the healthcheck is crucial for security.
+The default values for BONITA_HEALTHCHECK_USER and BONITA_HEALTHCHECK_PASSWORD are publicly known, and failing to update them can expose your healthcheck endpoint to unauthorized access and potential manipulation.
```
Comment (marcobonita) [inline modules/applications/pages/ui-builder/production-packaging.adoc]:
```suggestion
The default values for `BONITA_HEALTHCHECK_USER` and `BONITA_HEALTHCHECK_PASSWORD` are publicly known, and failing to update them can expose your healthcheck endpoint to unauthorized access and potential manipulation.
```

**Comment by akantcheff** [comment]
suggestion: we might add the variables `BONITA_HEALTHCHECK_USER` & `BONITA_HEALTHCHECK_PASSWORD` to the page [Docker container Environment variables](https://documentation.bonitasoft.com/bonita/2024.3/applications/ui-builder/ui-builder-docker-installation#environment-variables) ?

File: modules/applications/pages/ui-builder/ui-builder-docker-installation.adoc [hunk 2]
```diff
@@ -14,7 +14,8 @@ The following environment variable is mandatory:
 - `BONITA_API_URL`: The URL for the Bonita API.
 
 The following environment variables are optional:
-
+- `BONITA_HEALTHCHECK_USER`: Defines a custom username (default: `monitoring`).
+- `BONITA_HEALTHCHECK_PASSWORD`: Defines a custom password (default: `mon1tor1ng_adm1n`).
```
Comment (akantcheff) [inline modules/applications/pages/ui-builder/ui-builder-docker-installation.adoc]: It is a bit obvious with the names of those variables but this description is more accurate I guess:
```suggestion
- `BONITA_HEALTHCHECK_USER`: Defines the username to request the Bonita healthcheck API (default: `monitoring`).
- `BONITA_HEALTHCHECK_PASSWORD`: Defines the password to request the Bonita healthcheck API (default: `mon1tor1ng_adm1n`).
```

**Review by akantcheff — APPROVED** [review]

File: modules/applications/pages/ui-builder/ui-builder-docker-installation.adoc [hunk 3]
```diff
@@ -14,7 +14,8 @@ The following environment variable is mandatory:
 - `BONITA_API_URL`: The URL for the Bonita API.
 
 The following environment variables are optional:
-
+- `BONITA_HEALTHCHECK_USER`: Defines the username to request the Bonita healthcheck API (default: `monitoring`).
```
Comment (akantcheff) [inline modules/applications/pages/ui-builder/ui-builder-docker-installation.adoc:18]: The layout is broken :scream:  Add an empty line before the 1st element of the list

Comment (Shmayro) [reply modules/applications/pages/ui-builder/ui-builder-docker-installation.adoc:18]: good catch 👍

**Review by akantcheff — APPROVED** [review]

**Pull request merged**